src/components/comments/comments.js

```diff
@@ -9,6 +9,7 @@ const Comments = (props) => {
   };
   return (
     <div className="discussion-box">
+      <h3>Disqus comments</h3>
```

Comment:
I think we need to indicate these are the old ones
```suggestion
      <h3>⏳ Disqus comments</h3>
```